README.md

```diff
@@ -1,25 +1,20 @@
-# Macro-micro coupling with DuMuX and preCICE
-
-This repository showcases multi-scale coupling of [DuMuX](https://dumux.org/) simulations with [preCICE](https://www.precice.org/), using a two-scale coupled heat conduction problem in 2D as an examble. The DuMuX simulations can also be coupled with [their respective counterparts in Nutils](https://github.com/IshaanDesai/coupled-heat-conduction).
-
-
-Apart from DuMuX and preCICE, this project relies on three additional software components: the [DuMuX-preCICE adapter](https://github.com/precice/dumux-adapter/), the [preCICE Micro Manager](https://github.com/precice/micro-manager) and the (currently still private) DuMuX module [dumux-phasefield](https://git.iws.uni-stuttgart.de/dumux-appl/dumux-phasefield/).
+# Two-scale heat conduction with DuMuX and preCICE
 
+This repository showcases two-scale coupling of [DuMuX](https://dumux.org/) simulations with [preCICE](https://www.precice.org/), using a two-scale coupled heat conduction problem in 2D as an example. The DuMuX simulations can also be coupled with [their respective counterparts in Nutils](https://github.com/IshaanDesai/coupled-heat-conduction).
 
 ## Setup
 
-Being an example, this repository is sensitive to versioning, so make sure to get the correct versions of the involved software packages. **TLDR**s might be sufficient for quick setup, but they ignore all dependencies and will almost certainly fail or create conflicts for any system without previous configuration. Consulting the full setup instructions for each software component is very much recommended. 
+Being an example, this repository is sensitive to versioning, so make sure to get the correct versions of the involved software packages. **TLDR**s might be sufficient for quick setup, but they ignore all dependencies and will almost certainly fail or create conflicts for any system without previous configuration. Consulting the full setup instructions for each software component is very much recommended.
 
 After following the steps below, your repository structure should look like this:
 
-```
+```bash
 |-- dumux
-        |-- dumux
+    |-- dumux
         |-- dumux-adapter
-                        |--examples
-                                    |--macro-micro
-                                    ...
-                        ...
+            |--examples
+            |--macro-micro
```

Comment:
for now, `macro-micro` is still being set up in the examples folder (we could change this but would have to do so consistently

Reply:
Lets keep it in the examples folder for now, so basically not change this.